### PR TITLE
fix: remove `nuxi` dependency and prompt to install when needed

### PR DESCRIPTION
**Pull request by wattanx**
### 🔗 Linked issue
Fixes: https://github.com/nuxt/bridge/issues/900

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
In nuxt/cli, nuxt is set for bin field.
This may conflict with the nuxt command in nuxt 2.
https://github.com/nuxt/cli/blob/master/package.json#L13-L17

So, I'll change to install `nuxi` when you want to use nitro.

The following code was used as a reference.
https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/core/features.ts

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]
This looks good to me ❤️

File: packages/bridge/src/features.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  if (await resolvePackageJSON(name, { url: searchPaths }).catch(() => null)) {
+    return true
+  }
+
+  logger.info(`Package ${name} is missing`)
+  if (isCI) {
+    return false
+  }
+
+  const { confirm } = await prompts({
```
Comment (pi0) [inline packages/bridge/src/features.ts:21]: You can directly use `consola.prompt` to avoid new dependency

**Comment by pi0** [comment]
This looks another good idea but i believe safer approach might be supporting `nuxt2` binary directly from `nuxt@2` dist as a non conflicting feature. This looks more like a workaround to me.

Also, do we want to keep supporting nuxi for bridge? (we might potentially drop it in next nuxi major version).

**Comment by danielroe** [comment]
@pi0 Do you mean that users of Bridge would have to change the cli command in their package.json scripts to `nuxt2`, if they are not using `nitro`?

**Comment by pi0** [comment]
I am suggesting adding an additional (nonconflicting) `nuxt2` command so that the bridge can more safely enable nuxi by default.

**Comment by danielroe** [comment]
Sounds good to me. @wattanx are you happy (keeping all that this PR does) to also add a `nuxt2` legacy command?

**Comment by wattanx** [comment]
Yes, I'll add it!

**Comment by danielroe** [comment]
We should probably also update the docs too for the bridge migration.

**Pull request merged**